lib/grizzly/zwave/commands/configuration_set.ex

```diff
@@ -80,8 +80,14 @@ defmodule Grizzly.ZWave.Commands.ConfigurationSet do
     param_num = Command.param!(command, :param_number)
     size = Command.param!(command, :size)
     value = Command.param!(command, :value)
+    validate!(value, size)
     value_bin = <<value::signed-integer-size(size)-unit(8)>>
 
     <<param_num, size>> <> value_bin
   end
+
+  defp validate!(value, 1) when value in -128..127, do: :ok
+  defp validate!(value, 2) when value in -32768..32767, do: :ok
+  defp validate!(value, 4) when value in -2_147_483_648..2_147_483_647, do: :ok
+  defp validate!(_value, _byte), do: raise(ArgumentError, message: "Invalid parameter value")
```

Comment:
Can we make this message a little more descriptive? For example, "Invalid parameter value for configuration set: #{value}, size: #{size}". Just so if this gets logged we can get a little more context?

Reply:
Just saw your comment right after I posted!!! :)

Reply:
I am going to resolved mine as I think your message is worded better.